docs/edgeql/select.rst

```diff
@@ -843,6 +843,48 @@ shapes just like a non-computed link.
   } filter .name = "Iron Man";
 
 
+Using backlinks to find items
+-----------------------------
```

Comment:
```suggestion
Filtering on a known backlink
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

- Change the heading to more closely mirror our other "filtering" sections
- Change the heading level so that this falls under "Backlinks" instead of adjacent to it